splunk-otel-collector.yaml

```diff
@@ -40,6 +40,12 @@ pipeline:
       cp cmd/otelcol/config/collector/ecs_ec2_config.yaml ${{targets.contextdir}}/etc/otel/collector/ecs_ec2_config.yaml
 
 subpackages:
+  - name: ${{package.name}}-compat
+    description: Symlink for the otel binary in the home folder
```

Comment:
This is at root, not in home

Reply:
@EyeCantCU This was a mixup. Upstream expects the binary in the root directory. Fixed the description